src/constants.ts

```diff
@@ -0,0 +1,80 @@
+export const TOKEN_SYMBOLS_MAP = {
+  ACX: {
+    name: "ACX",
+    symbol: "ACX",
+    decimals: 18,
+    mainnetAddress: "0x44108f0223A3C3028F5Fe7AEC7f9bb2E66beF82F",
+  },
+  BAL: {
+    name: "Balancer",
+    symbol: "BAL",
+    decimals: 18,
+    mainnetAddress: "0xba100000625a3754423978a60c9317c58a424e3D",
+  },
+  BADGER: {
+    name: "Badger",
+    symbol: "BADGER",
+    decimals: 18,
+    mainnetAddress: "0x3472A5A71965499acd81997a54BBA8D852C6E53d",
+  },
+  BOBA: {
+    name: "Boba",
+    symbol: "BOBA",
+    decimals: 18,
+    mainnetAddress: "0x42bBFa2e77757C645eeaAd1655E0911a7553Efbc",
+  },
+  DAI: {
+    name: "Dai Stablecoin",
+    symbol: "DAI",
+    decimals: 18,
+    mainnetAddress: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+  },
+  ETH: {
+    name: "Ether",
+    symbol: "ETH",
+    decimals: 18,
+    mainnetAddress: "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
+  },
+  MATIC: {
+    name: "Matic",
+    symbol: "WMATIC",
```

Comment:
MATIC/WMATIC is an odd one, but the symbol is MATIC on Ethereum mainnet and WMATIC on Polygon.

Do we rely on it being called WMATIC when we are referring to the Ethereum mainnet variant?

Reply:
Hmm good catch. I think this should be `MATIC` yea

Reply:
Changed here https://github.com/across-protocol/sdk-v2/pull/141/commits/c96a884bc1daf827578340fb87f5e2b36bc5383a#diff-8fa4b52909f895e8cda060d2035234e0a42ca2c7d3f8f8de1b35a056537bf199R85